website/docs/d/spring_cloud.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud"
+description: |-
+  Gets information about an existing Spring Cloud Service
+---
+
+# Data Source: azurerm_spring_cloud
+
+Use this data source to access information about an existing Spring Cloud Service.
+
+## Example Usage
+
+```hcl
+data "azurerm_spring_cloud" "example" {
+  name                = azurerm_spring_cloud.example.name
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+output "spring_cloud_id" {
+  value = "${data.azurerm_spring_cloud.example.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Spring Cloud Service resource.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Spring Cloud Service exists.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of Spring Cloud Service.

```

Comment:
if this is labelled as a `Spring Cloud Service` should that be the resource name here?